### ISSUE TITLE
Log flooded with 'VOC Density' exceeded maximum of 1000.

### ISSUE THREAD

**Issue by xszuflax** [tomekkleszcz/homebridge-electrolux-devices#40]
**Describe The Bug:**
Home bridge log flooded with messages:

```
[03/03/2024, 10:05:35] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1184 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 10:05:35] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1343 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 10:06:05] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1148 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[03/03/2024, 10:06:05] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1290 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
```

* **Plugin Version**: 0.0.8
* **Homebridge Version**:  Homebridge v1.7.0  UI v4.55.1
* **Node.js Version**: 20.11.1
* **Operating System**: Raspbian  hb-service 

I believe this was introduced sometime between 0.0.6 and 0.0.8

**Comment by tomekkleszcz** [comment]
Hi @xszuflax , I'll limit the maximum value for now, but I believe it would be fixed as soon as the TVOC would be converted to VOC density. I'll update you on the progress.

**Comment by tomekkleszcz** [comment]
Should be resolved in #42

**Comment by xszuflax** [comment]
Yep,

it looks better now. Thank you :-)

**Comment by xszuflax** [comment]
I'm afraid it has not be fixed after all:

```
[04/03/2024, 06:17:22] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2284 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:17:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2284 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:18:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2284 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:18:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2284 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:19:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:19:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:20:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:20:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2284 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:21:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:21:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:22:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:22:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:23:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:23:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:24:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:24:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2253 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:25:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2222 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:25:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2222 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:26:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2222 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:26:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2180 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:27:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2114 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:27:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2083 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:28:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2020 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:28:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2194 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:29:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2159 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:29:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2124 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:30:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2124 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:30:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2089 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:31:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2058 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:31:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1933 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:32:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2048 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:32:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2017 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:33:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1991 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:33:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 2048 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:34:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1955 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:34:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1955 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:35:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1924 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:35:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1862 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:36:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1828 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:36:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1945 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:37:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1919 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:37:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:38:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:38:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:39:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:39:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:40:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:40:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1863 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:41:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1831 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:41:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1831 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:42:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1776 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:42:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1741 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:43:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1667 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:43:50] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1739 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:44:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1715 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:44:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1684 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:45:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1715 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:45:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1684 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:46:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1636 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:46:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1604 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:47:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1583 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:47:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1542 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:48:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1519 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:48:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1500 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:49:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1500 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:49:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1500 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:50:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1446 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:50:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1427 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:51:20] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1401 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:51:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1401 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:52:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1544 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:52:50] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1493 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:53:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1493 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:53:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1476 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:54:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1451 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:54:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1431 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:55:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1382 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:55:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1354 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:56:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1302 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:56:38] [Homebridge UI] Running scheduled instance backup...
[04/03/2024, 06:56:38] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-61g7yj/homebridge-backup-0E6D4663E234.tar.gz
[04/03/2024, 06:56:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1282 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:57:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1282 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:57:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1258 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:58:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1238 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:58:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1215 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:59:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1343 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 06:59:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1295 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:00:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1295 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:00:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1295 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:01:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1247 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:01:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1222 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:02:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1201 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:02:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1175 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:03:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1157 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:03:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1157 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:04:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1124 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:04:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1090 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:05:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1069 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:05:49] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1024 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
[04/03/2024, 07:06:19] [homebridge-electrolux-devices] This plugin generated a warning from the characteristic 'VOC Density': characteristic was supplied illegal value: number 1024 exceeded maximum of 1000. See https://homebridge.io/w/JtMGR for more info.
```

I'm on v 0.0.8

**Comment by mafredri** [comment]
It seems like two changes from my PR (#18) didn’t get included in #42: 

- https://github.com/tomekkleszcz/homebridge-electrolux-devices/pull/18/commits/9b3eebec468c0d74ad12fb4ea39b9dd1587d1275#diff-ca571ea2be90550f71f8f284baa67c25978327986ebdadd3a11faa4e475a211eR261 (Math.min)
- https://github.com/tomekkleszcz/homebridge-electrolux-devices/pull/18/commits/3b2883d979c8d6d72f703141881db8c41f53170d

**Comment by xszuflax** [comment]
Hello @tomekkleszcz, after 24h it looks good I will consider this fixed now :-) 

Thank you very much!
